service/models.py

```diff
@@ -21,7 +21,7 @@ class OrderItem(db.Model):
     # Order Item Table Schema
     ##################################################
     item_id = db.Column(db.Integer, primary_key=True)
-    product = db.Column(db.String(40), nullable=False)
```

Comment:
@rutvik2151 It would be better to rename the column to ```product_id```.

Reply:
Will do.